user-interface/src/main/java/life/qbic/datamanager/parser/MetadataConverter.java

```diff
@@ -327,6 +342,7 @@ public String propertyName() {
 
   enum NGSMeasurementProperty {
     MEASUREMENT_ID("measurement id"),
+    SAMPLE_CODE("sample code"),
```

Comment:
This property does not exist in the template sheets for ngs registration and editing, which leads to the issue that no measurements can be registered or edited.
 Did you mean `QBiC Sample Id` ?

Reply:
Yes I did. The issue with the wrong properties being used seems to have been in the code already. It should be fixed now.